### ISSUE TITLE
Empty Library view, design improvement

### ISSUE THREAD

**Issue by llemeurfr** [readium/readium-desktop#537]
When opening Thorium with an empty database, the screen that appears contains a huge cross, and About Thorium on the top left. This can be improved. 

The cross is too intrusive, let's replace it with a dotted rectangle as in many other apps. 
Also, "About Thorium" must be moved below the level of the import area. 
The text can also be improved.

**Comment by llemeurfr** [comment]
![Capture d'écran 2019-08-08 09 39 23](https://user-images.githubusercontent.com/14943186/62684866-dc751c80-b9c1-11e9-8924-7c132829a780.png)

**Comment by llemeurfr** [comment]
Here is what we'll do. 

<img width="617" alt="Capture d'écran 2019-08-08 10 08 59" src="https://user-images.githubusercontent.com/14943186/62686230-91a8d400-b9c4-11e9-811c-e04e538fdb06.png">

**Comment by llemeurfr** [comment]
Label translations: 

FR : 
Déposez vos fichiers EPUB dans cette fenêtre
ou bien utilisez le bouton "+" ci-dessus.
Les fichiers protégés par LCP sont pris en charge. 

DE :
Drop EPUB-Dateien in diesem Fenster
oder benutzen Sie die taste "+" oben.
LCP-geschützte Dateien werden unterstützt.

**Comment by llemeurfr** [comment]
This is a spin off of #278.

**Comment by llemeurfr** [comment]
I have already prepared the label in english (noPublicationHelp) and slightly modified labels: 

"Drop ebooks in this window or use the (+) button above. LCP protected files are supported."

I was careful to avoir the " sign in json, just in case transform to HTML goes wrong. 

I'd like the text to be on 3 lines with the last a bit smaller, but I had to use the unique property I had. I suppose properties must be updated to noPublicationHelpL1, noPublicationHelpL2 and noPublicationHelpL3.
